src/commands/music-commands/seek.ts

```
@@ -1,86 +1,95 @@
-            if (isNaN(seekTime) || seekTime < 0 || seekTime >= currentDuration / 1000) {
-                return harmonix.client.createMessage(msg.channel.id, {
-                    embed: {
-                        color: 0xFF0000,
-                        title: `Error | You may seek from 0 - ${Math.floor(currentDuration / 1000)} seconds`
-                    }
-                });
-            }
+      const seekTime = Number(args[0]);
+      const currentDuration = player.queue.current.duration || 0;
+      if (
+        isNaN(seekTime) ||
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)